addOns/fuzz/src/main/java/org/zaproxy/zap/extension/fuzz/httpfuzzer/HttpFuzzerHandler.java

```diff
@@ -239,15 +240,15 @@ public HttpMessage getMessage(MessageContainer<HttpMessage> messageContainer) {
     }
 
     @SuppressWarnings("unchecked")
-    protected <T1 extends HttpFuzzerMessageProcessor, T2 extends HttpFuzzerMessageProcessorUI<T1>>
+    public <T1 extends HttpFuzzerMessageProcessor, T2 extends HttpFuzzerMessageProcessorUI<T1>>
```

Comment:
Why is this (and following) being changed?

Reply:
I think these are not needed now. will change it.